### ISSUE TITLE
Live Performance

### ISSUE THREAD

**Issue by adam3smith** [citation-style-language/zotero-bits#55]
http://forums.zotero.org/discussion/27369/feature-request-adding-an-item-type-for-live-performances/#Item_3

Examples:
MLA:
Title. By Name of Author/Composer. Dir. Name of Director. Perf. Names
          of Performers. Theatre, City. Date. Performance.

Hamlet. By William Shakespeare. Dir. John Gielgud. Perf. Richard
          Burton. Shubert Theatre, Boston. 4 Mar. 1964. Performance.

Monash Harvard:
Bell, J (dir.) 2011, Much ado about nothing, by W Shakespeare, theatre performance, 8 April- 14 May 2011, Bell Shakespeare Company, Drama Theatre, Sydney Opera House, viewed 10 April 2011.

CMoS:
1. Jan Kurro, "Songs of Sondheim" (vocal performance, Her Majesty’s Theatre, Ballarat, May 6, 2008).

It _might_ be possible to get close to this using Presentation/Event and expanding creator roles, but for MLA that might not do.

**Comment by bwiernik** [comment]
It looks like this could be handled in MLA fine with `speech` and expanded creator roles for writer, performer.

**Comment by dstillman** [comment]
Re: https://forums.zotero.org/discussion/comment/451454/#Comment_451454, happy to add this anytime (as we did Preprint and Standard) if we know the fields we want.

**Comment by aarblaster** [comment]
Thanks @dstillman. This is definitely a challenge. Presentation/event is probably the closest currently and what I've been modifying. I'm researching theatre lighting design and we use [Harvard (York)](https://subjectguides.york.ac.uk/referencing-style-guides/harvard). 
In text plays (live performance) are rendered as:

> (Lynn & Jay, 2012)

In reference lists:

> Lynn, J. & Jay, A. (2012, April 16). Yes, Prime Minister [Play]. Theatre Royal, York. Performers: Graham Seed and Michael Simkins. Director: Jonathan Lynn.  

My understanding is the fields are as follows:
Author. (date). Title [performance-type], City. Performers/Creative Team.

The order and addition of the creative team does change depending on the specialisation. As does the names of the performers. The director is always included.
The following fields are useful and would work for MLA and Monash Harvard as identified by @adam3smith:

- Title:
- Performance Type:
- Author:
- Venue:
- City:
- Date:
- Company:
- Director:
- Set Designer:
- Lighting Designer:
- Sound Designer:
- Performer:
- Composer:
- Accessed:
- URL:

These fields could then be used for other live performance types as needed (see the musical performance (live) reference for Harvard (York) as well).

**Comment by bwiernik** [comment]
What roles do Lynn and Jay play in that production? Lynn appears to be the director--is Jay the playwright?

**Comment by adam3smith** [comment]
In terms of mapping from Zotero to CSL, 
itemType: performance
performanceType:  genre
date: event-date
author, director, performer, composer, accessed, URL, DOI all as named

I'm a bit skeptical about the extent to which set, lightning, and sound design should be part of the Zotero data model -- we just can't reasonably include the whole cast (e.g., for a musical, music director & choreographer would likely be more important than the designers etc.).

The one tricky question I see is venue & city. In CSL we only have `event-place`. I see why one would want to store (and potentially cite) these separately, but we don't have anything good in the data model and I'm wary of adding for just that purpose, so I think we'll either want to combine these into a single Zotero field or store separate fields and _output_ those comma-delimited to `event-place` -- where users didn't like that, they could just omit one of the two.

@bwiernik Lynn & Jay wrote Yes, Prime Minister, i.e. both are authors.

**Comment by bwiernik** [comment]
I agree with the comma-delimited output of venue and city together to `event-place`

Another option would be to map venue to `event-title`, but that's not a great conceptual fit and I would prefer to reserve `event-title` for something like a festival where a performance occurred.

**Comment by adam3smith** [comment]
Since we're talking new item types, here's the proposal with mapped CSL types
   
    itemType: performance
    Title: title
    Author: author
    Director: director
    Performer: performer
    Composer: composer
    Performance Type: genre
    Venue: event-place
    City: event-place (see above)
    Date: event-date
    Company: publisher
    Accessed: accessed
    URL: url
    DOI: DOI
    Citekey: citation-key
    Archive: archive
    Loc. in Archive:  archive_location
    
I'd like to start adding better archive fields to all item types so also would suggest

    Archival collection: archive_collection
    Archive location: archive_place

Not sure those are the right Zotero labels yes. 

updated based on conversation below.

**Comment by bwiernik** [comment]
@adam3smith There is a separate CSL type `event` that's intended for exhibitions and workshops. I think they should be separate Zotero types 

For Company, maybe CSL publisher? From what I see, they seem to be cited like a publisher, for example https://www.tandfonline.com/doi/full/10.1080/17450918.2012.679301

**Comment by adam3smith** [comment]
Agreed on not cramming exhibitions in there, thanks.

For company/publisher -- the linked article cites the RSC as a publisher for their web content, not for their performances, but I guess I could be convinced this is right for performances: people would think of "The RSC 2021 production of Hamlet" or "The Alvin Ailey 1970 production of Roaratorio" etc.

**Comment by adam3smith** [comment]
Updated the proposal above -- would be good if someone else looks at this, but looks good from my end.

**Comment by bwiernik** [comment]
Here's an example in a journal that publishes theater reviews--I think publisher still works well there https://extra.shu.ac.uk/emls/11-1/revkwtn.html

**Comment by bwiernik** [comment]
Do we want to include a field for something like a festival or collection of performances? Something like: 

Festival (not sure on a good broader label) : `event-title`


Should probably also include Series for something like award or honorary performances:

Series : `collection-title`

**Comment by aarblaster** [comment]
Thanks for all the work here.
@adam3smith the mapping to CSL seems good to me, and I'm a big fan of adding the archive bits.
My only question is about designers. Performances in theatre, opera and dance often use the trinity of designer and director and company as the key identifier. This is because productions may be remounted with one or more changed, thereby enabling comparisons. 
For example a director may direct Macbeth for the RSC on multiple occasions, but the designer might be different each time.
Designer in this context would ordinarily be set and or costume but calling it `Designer` would provide scope for flexibility.

**Comment by aarblaster** [comment]
@bwiernik I think adding festival is a good idea. The label of festival seems a good way of describing multiple collections to me. Sydney Festival or Edinburgh Fringe being good examples.
I am a little confused about `series` and what you mean by honorary performances?

**Comment by bwiernik** [comment]
I think let's just call the field Event or Event Name (mapped to `event-title`)

On `series`, for example, if I wanted to cite a performance that was part of this special series for this orchestra (http://www.heritagephilharmonic.org/musician-showcase-series.html) or a performance that is part of the Broadway at the White House series (eg, [Broadway at the White House](https://www.youtube.com/watch?v=xlET8mUe3Ow))
